plugins/main/public/controllers/management/components/management/cluster/cluster-overview.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { compose } from 'redux';
+import {
+  withErrorBoundary,
+  withGlobalBreadcrumb,
+  withReduxProvider,
+  withUserAuthorizationPrompt,
+} from '../../../../../components/common/hocs';
+import { cluster } from '../../../../../utils/applications';
+import { AppState, WzRequest } from '../../../../../react-services';
+import { ClusterDisabled } from '../../../../../components/management/cluster/cluster-disabled';
+import { ClusterDashboard } from '../../../../../components/management/cluster/dashboard/dashboard';
+
+interface ClusterOverviewState {
+  clusterEnabled: boolean;
+  isClusterRunning: boolean;
+  statusRunning: string;
+}
+
+export const ClusterOverview = compose(
+  withErrorBoundary,
+  withReduxProvider,
+  withGlobalBreadcrumb([{ text: cluster.breadcrumbLabel }]),
+  withUserAuthorizationPrompt([
+    { action: 'cluster:status', resource: '*:*:*' },
+  ]),
+)(
+  class ClusterOverview extends Component<unknown, ClusterOverviewState> {
+    _isMount = false;
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        clusterEnabled: false,
+        isClusterRunning: false,
+        statusRunning: 'no',
+      };
+    }
+
+    async componentDidMount() {
+      this._isMount = true;
+      const clusterEnabled =
+        AppState.getClusterInfo() &&
+        AppState.getClusterInfo().status === 'enabled';
+
+      const status: any = await WzRequest.apiReq('GET', '/cluster/status', {});
```

Comment:
**thought**: I am not sure because we use the value stored in the cookie to know if the cluster is enabled, and we do an API request that has the more updated value.